git_flow.rst

```diff
@@ -0,0 +1,189 @@
+.. -*- mode: rst -*-
+
+.. role:: bash(code)
+   :language: bash
+
+Recommended GitHub workflow
```

Comment:
You could always make a separate page for that too. I found this more helpful than most of the high level git walk-throughs.